src/Alert.jl

```diff
@@ -97,13 +97,35 @@ You can clear the cache for this tenant using [`mPulseAPI.clearAlertCache`](@ref
 """
 function getRepositoryAlert(token::AbstractString; alertID::Int64=0, alertName::AbstractString="")
 
-    alert = getRepositoryObject(
+    alert_list = getRepositoryObject(
                 token,
                 "alert",
-                Dict{Symbol, Any}(:id => alertID, :name => alertName)
+                Dict{Symbol, Any}(:id => alertID, :name => alertName),
+                filterRequired=false
         )
 
-    return alert
+    # Always convert to an array for easier processing
+    if !isa(alert_list, Array)
+        alert_list = [alert_list]
```

Comment:
Specify array type

Reply:
the same logic is done in the file for statmodel and domain, so specified there as well